tests/auxil/asyncio.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
```

Comment:
is it a good idea to name this file as `asyncio`? Maybe `asyncio_extension`/`async_helpers`